cmd/ocm-support/sync-cloud-resources/cmd.go

```diff
@@ -0,0 +1,209 @@
+package sync_cloud_resources
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+
+	"github.com/spf13/cobra"
+)
+
+var Cmd = &cobra.Command{
+	Use:          "sync-cloud-resources [branch-name] [csv-path]",
+	Short:        "Syncs cloud resources in AMS and generates quota rules for them",
+	Long:         "Syncs cloud resources in AMS and generates quota rules for them",
+	RunE:         syncCloudResources,
+	SilenceUsage: true,
+	Args:         cobra.ExactArgs(2),
+}
+
+var args struct {
+	dryRun bool
+}
+
+func init() {
+	flags := Cmd.Flags()
+	flags.BoolVar(&args.dryRun, "dry-run", true, "If false, it commits the generated cloud resources and quota rule changes to the remote branch at https://gitlab.cee.redhat.com/service/uhc-account-manager")
```

Comment:
I will leave that decision to you, but IMO, this can be False by default

Reply:
Keeping it as true here since we don't want users to accidentally run the command. Also, this is the consistent behavior across the subcommands in this tool, e.g. https://github.com/openshift-online/ocm-support-cli/blob/main/cmd/ocm-support/patch/account/cmd.go#L35